### PR TITLE
*: Post mod-log info improvements

### PR DESCRIPTION
**Pull request by Chiiruno**
Addresses #858

### CODE REVIEW

File: go/src/meguca/db/bans.go [hunk 1]
```diff
@@ -121,52 +127,54 @@ func Ban(board, reason, by string, expires time.Time, log bool, ids ...uint64) (
...
-			err = auth.ModLogPost(post.id, post.op, mLog)
-
-			if err != nil {
-				return err
-			}
-
-			err = common.BanPost(post.id, post.op)
-			if err != nil {
-				return err
+		if (len(modLog) == len(posts)) {
```
Comment (Chiiruno) [inline go/src/meguca/db/bans.go]: Should I have it return an error in an else block if this is not true?
Shouldn't this always be true?
Should I remove this check, then?

**Comment by bakape** [comment]
Okay. This is becoming too huge of a mess. I'll take what we have  working and rewrite the system, once I'm done with the current hydron changes. 

superseded by #862

**Comment by Chiiruno** [comment]
@bakape Do you need the current code to base anything off of, or is it alright if I go ahead and delete the branch?

**Comment by bakape** [comment]
I'm working against master. You can delete it.

**Pull request closed without merging**